Packs/TaniumThreatResponse/Integrations/TaniumThreatResponse/TaniumThreatResponse.yml

```diff
@@ -3,14 +3,6 @@ commonfields:
   id: Tanium Threat Response
   version: -1
 configuration:
-- display: Fetch incidents
-  name: isFetch
-  required: false
-  type: 8
-- display: Incident type
-  name: incidentType
-  required: false
-  type: 13
```

Comment:
Why did you move this part down? I think the previous order was better.

Reply:
I did not intentionally move this part down, it was likely due to making changes through the Integration Settings UI